### ISSUE TITLE
State tables overflowing on mobile 

### ISSUE THREAD

**Issue by olivierlacan** [COVID19Tracking/website#208]
![BE6E914E-BA22-4B1F-80CE-24E7D41135B8](https://user-images.githubusercontent.com/65950/77699930-a523a200-6f89-11ea-9e76-2544115a27a7.png)

**Comment by hal2814** [comment]
<img width="317" alt="Screen Shot 2020-03-29 at 12 31 33 PM" src="https://user-images.githubusercontent.com/30516063/77858668-a804d980-71b9-11ea-89b4-7188872763b7.png">

This appears to have been fixed/changed in the current build from the screenshots I see in this issue. Should this issue be closed?

**Comment by beep** [comment]
_(Cross-posting from Slack.)_

I did a quick riff on this using [tablesaw](https://github.com/filamentgroup/tablesaw/), a `responsive tables` library from Filament Group that I’ve used in the past.

https://codepen.io/beep/pen/57ec7b1b9c4895edad60f0bd768cb574

To get this to work, I had to tweak the table structure a little:

1. I had to ditch the empty cells in the first header row, move the date/deaths/total cells up to the first row, and make them `rowspan="2"`. (Basically, the library looks for headers in the first row of the `thead`.)
2. I manually assigned `data-tablesaw-priority` values to the other top-level header cells.

Also, if the table’s in a `tablesaw-overflow` wrapper, it’ll automatically overflow if it needs to.
